ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/Main.scala

```diff
@@ -29,17 +29,26 @@ import com.daml.resources.{ProgramResource, ResourceOwner}
 import scopt.OptionParser
 
 object Main {
+  private val runnerName = "In-Memory Ledger"
```

Comment:
Scala people like newlines.

Also constants are supposed to be capitalized. (I forget this a lot too.)

```suggestion
  private val RunnerName = "In-Memory Ledger"

```